style.css

```diff
@@ -121,7 +121,7 @@ body {
 	-moz-box-sizing: border-box;
 	box-sizing: border-box;
 	color: #000;
-	font-family: Inter, -apple-system, BlinkMacSystemFont, "Helvetica Neue", Helvetica, sans-serif;
+	font-family: -apple-system, BlinkMacSystemFont, "Helvetica Neue", Helvetica, sans-serif;
```

Comment:
Should we be replacing "Inter" with "Inter var", instead of just removing it?

Reply:
We include Inter var within the `@supports` media query in the next CSS rule, which only targets browsers that supports variable fonts.